src/frontends/common/include/common/extensions/decoder_transformation_extension.hpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
BTW, `extensions/decoder_transformation_extension.hpp ` => `extensions/decoder_transformation.hpp` to exclude double `extension` name from path?

Looks like we are trying to use this scheme in frontends, but it's just are thoughts

Reply:
CC @ilyachur @slyalin how do you think?

Reply:
extensions -> extension
{name}_extension -> {name}
fixed